app/routes/users/components/UserConfirmation.js

```diff
@@ -98,16 +103,19 @@ const UserConfirmation = ({
           />
           <RadioButtonGroup name="gender">
             <Field
+              name="genderMale"
```

Comment:
These names are determined by the RadioButtonGroup's `name="gender"` and should be removed or set explicitly to `name="gender"`. They refer to the field `gender` that gets posted to the backend, and redux-form uses the `name` variable to be the same to ensure that the radiobuttons switches correctly.